src/GUI/EFCorePowerTools/Handlers/ReverseEngineerHandler.cs

```diff
@@ -148,12 +148,12 @@ public async Task ReverseEngineerCodeFirstAsync(Project project)
                         preselectedTables.AddRange(normalizedTables);
                     }
 
-                    for (var i = 0; i < predefinedTables.Count; i++)
-                    {
-                        var preselectedTable = preselectedTables.FirstOrDefault(c => c.Name.Equals(predefinedTables[i].Name, StringComparison.OrdinalIgnoreCase));
-                        if (preselectedTable != null)
-                            predefinedTables[i] = preselectedTable;
-                    }
+                    //for (var i = 0; i < predefinedTables.Count; i++)
```

Comment:
Why is this commented out?

Reply:
That code is no longer needed because now TableModel object properties are splattered into view model. When the user confirms his selection, the view model properties are used to create new TableModel instances that reflects user selection. Before this change, the TableInformationModel instance was directly used and returned by view models so I had to pass to them the correct TableInformationModel instance (the one coming from db table list or the one coming from preselected table list).

Reply:
Then why not just delete it?

Reply:
Good question, my stupidity is the answer. I'll delete it